.github/workflows/nightly.yml

```diff
@@ -113,3 +114,53 @@ jobs:
           alr-nightly-*.zip
         tag: nightly
         rm: false
+
+  build-macos-universal:
+    runs-on: macos-latest
+    needs: [build]
+    steps:
+      - name: Install Python 3.x (required for releaser)
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+
+      - name: download x86_64
+        uses: robinraju/release-downloader@v1
+        with:
+          fileName: '*-x86_64-macos.zip'
+          latest: true
+          preRelease: true
+          out-file-path: 'x86_64'
+          extract: true
+      - name: check x86_64 attributes
+        run: xattr -vr x86_64
```

Comment:
Is this a debug leftover? Or missing in `ci-macos`?

Reply:
Yes, it's a debug thing -- well, really investigative; since the download is via a url, I was worried that the `com.apple.quarantine` attribute might get applied to the download contents, which would mean that when a user downloads the .zip via curl they would still have to remove the attribute even though curl doesn't apply it itself. Result of investigation was that there's no quarantine attribute; will remove this step.